src/test/scala/com/cognite/sdk/scala/common/WritableBehaviors.scala

```diff
@@ -13,71 +13,87 @@ trait WritableBehaviors extends Matchers { this: FlatSpec =>
       idsThatDoNotExist: Seq[PrimitiveId],
       supportsMissingAndThrown: Boolean
```

Comment:
We should have an additional flag, `supportsDeletes`, or possibly `deletable: DeleteByIds[Id, PrimitiveId]` which most often would be the same as `writable` but do nothing in the case of data sets.
We'll have to remove `with DeleteByIds[Id, PrimitiveId]` since we can't require that from the writable resource if we no longer require deletes.